imports/plugins/included/taxes-avalara/server/methods/taxCalc.js

```diff
@@ -27,12 +27,12 @@ taxCalc.getPackageData = function () {
  */
 function getUrl() {
   const packageData = taxCalc.getPackageData();
-  const { productionMode } = packageData.settings.avalara;
+  const { mode } = packageData.settings.avalara;
   let baseUrl;
-  if (!productionMode) {
+  if (!mode) {
     baseUrl = "https://sandbox-rest.avatax.com/api/v2/";
   } else {
-    baseUrl = "https://rest.avatax.com";
+    baseUrl = "https://rest.avatax.com/api/v2/";
```

Comment:
It is always better (more understandable and faster code) to do fewer `!`, so you should flip this `if` logic:

```js
if (mode) {
  baseUrl = "https://rest.avatax.com/api/v2/";
} else {
  baseUrl = "https://sandbox-rest.avatax.com/api/v2/";
}
```